frontend/test/metabase/scenarios/models/model-actions.cy.spec.js

```diff
@@ -154,6 +154,34 @@ describe("scenarios > models > actions", () => {
     );
   });
 
+  it("should allow to create an action with the New button", () => {
```

Comment:
One unrelated note, I just realized that even if this file is relatively new, it wasn't created in TypeScript 👁️ 👄 👁️ .

Reply:
I don't think any of our cypress files are in typescript. setting up types for cypress tests will be task in itself.

Reply:
@iethree crap yeah this is Cypress 🤦 . My brain stop working on Friday evening aparently.